src/components/atoms/Title.js

```diff
@@ -6,12 +6,13 @@ import PropTypes from "prop-types";
  */
 
 export function Title(props) {
-  return <h1>{props.children}</h1>;
+  return <h1 id={props.id}>{props.children}</h1>;
```

Comment:
So I see you've added an id element here to be able to get the element in cypress. However, are we expected to do this for every element then? Could you not use the tag as the selector rather than setting an id ?